content/docs/dvclive/api-reference/live/index.md

```diff
@@ -50,10 +46,16 @@ Its parameters are focused on configuring the behavior of subsequent calls to
 - `resume` - If `True`, DVCLive will try to read the previous `step` from the
   `path` directory and start from that point. _Default_: `False`.
 
+  ⚠️ Don't change the default value unless you are going to update the `step`
+  value during training.
+
 - `summary` - If `True`, upon each `Live.log()` call, DVCLive will generate a
-  _metrics summary_ (usable by `dvc metrics`). The _summary_ will be located at
+  summary (usable by `dvc metrics`). The summary will be located at
   `{path}.json`. _Default_: `True`.
 
+  ⚠️ Don't change the default value unless you are going to update the `step`
+  value during training.
```

Comment:
Also could explain more here: `If you are not using `steps`, don't set to `False` since the _summary_ will be the only record of your metrics.`

Reply:
Added but with some rephrasing please TAL